app/cdap/components/PipelineHistory/PipelineHistoryTableRow.tsx

```diff
@@ -104,7 +104,8 @@ export const PipelineHistoryTableRow = ({
 
   const isLatest = appVersion === latestVersion;
   const dateLabel = isLatest ? T.translate(`${PREFIX}.latest`) : T.translate(`${PREFIX}.older`);
-  const displayedDesc = description.length > 190 ? description.slice(0, 190) + '...' : description;
+  const displayedDesc =
+    description && description.length > 190 ? description.slice(0, 190) + '...' : description;
```

Comment:
nit: description?.length

Reply:
Also displayedDesc is empty when description is empty ?

Reply:
nice catch. yeah I think when description, displayedDesc is also empty.